packages/block-library/src/post-date/index.php

```diff
@@ -48,15 +48,27 @@ function render_block_core_post_date( $attributes, $content, $block ) {
 
 	$wrapper_attributes = get_block_wrapper_attributes( array( 'class' => implode( ' ', $classes ) ) );
 
+	$prefix = '';
+	if ( isset( $attributes['prefix'] ) && $attributes['prefix'] ) {
+		$prefix = '<span class="wp-block-post-date__prefix">' . $attributes['prefix'] . '</span>';
+	}
+
 	if ( isset( $attributes['isLink'] ) && $attributes['isLink'] ) {
 		$formatted_date = sprintf( '<a href="%1s">%2s</a>', get_the_permalink( $post_ID ), $formatted_date );
 	}
 
+	$suffix = '';
+	if ( isset( $attributes['suffix'] ) && $attributes['suffix'] ) {
+		$suffix = '<span class="wp-block-post-date__suffix">' . $attributes['suffix'] . '</span>';
+	}
+
 	return sprintf(
-		'<div %1$s><time datetime="%2$s">%3$s</time></div>',
+		'<div %1$s>%2$s<time datetime="%3$s">%4$s</time>%5$s</div>',
```

Comment:
I think prefix and suffix would need `esc_html`. A contributor user could add scripts there.

Reply:
It would need to be `wo_kses_post` because the rich text string may contain formats such as bold / italics etc.

Reply:
Applied `wp_kses_post` to the Prefix and Suffix for the Post Date block output https://github.com/WordPress/gutenberg/pull/61920/commits/4d1ca75f6223cee84f742d48d7ddea8fa286ad0b